test/pls.jl

```diff
@@ -327,8 +327,8 @@ end
 
     @testset "zerocorr PCA" begin
         @test length(fmnc.rePCA) == 1
-        @test fmnc.rePCA.subj == [0.5, 1.0]
-        @test MixedModels.PCA(fmnc).subj.loadings == I(2)
+        @test fmnc.rePCA.subj ≈ [0.5, 1.0]
+        @test_broken MixedModels.PCA(fmnc).subj.loadings ≈ I(2)
```

Comment:
@palday Could you take a look at this, please?  `git blame` thinks you added this testset and I am not sure the results are well-defined but I don't know enough about what you wanted to have happen.

Reply:
I think the thought process there was "zerocorr models should already have an orthogonal variance-covariance matrix, so the PC rotation should be the identity", but it occurs to me that that may not be true.

Reply:
Ah, of course, it's only true up to a permutation of the columns.